era5cli/cli.py

```diff
@@ -128,6 +128,17 @@ def _build_parser():
                              ''')
     )
 
+    common.add_argument(
+        "--dryrun", type=_str2bool, default=False,
+        help=textwrap.dedent('''\
+                             Whether to start downloading the request or just
+                             print information on the chosen parameters and
+                             output file names. "--dryrun False" starts
```

Comment:
Maybe mention that "--dryrun True" will print the information instead of mentioning the False option, since the default is False already.